apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -207,6 +208,22 @@ def nnz(self) -> int:
         self._verify_open_for_reading()
         return cast("SparseNDArrayWrapper", self._handle).nnz
 
+    def delete_cells(self, coords: options.SparseNDCoords, *, platform_config: PlatformConfig | None = None) -> None:
+        """Deletes cells at the specified coordinates in a :class:`SparseNDArray`.
+
+        Args:
+            coords:
+                A per-dimension ``Sequence`` of scalar, slice, sequence of scalar or
+                `Arrow IntegerArray <https://arrow.apache.org/docs/python/generated/pyarrow.IntegerArray.html>` values
+                defining the region to read.
+        """
+        if isinstance(platform_config, (TileDBCreateOptions, TileDBWriteOptions)):
```

Comment:
this test should check TileDBDeleteOptions directly, not indirectly rule out other types.  Consider what happens if we add another (unacceptable) type in the API.

E.g., use:
> if not isinstance(platform_config, TileDBDeleteOptions): ...